integration_test/third_party_apps_data/applications/cassandra/debian_ubuntu/install

```diff
@@ -1,7 +1,20 @@
 set -e
 
 # Required to install cassandra
-echo "deb http://www.apache.org/dist/cassandra/debian 22x main" | sudo tee -a /etc/apt/sources.list.d/cassandra.sources.list
+source /etc/os-release
+if [[ "${VERSION_CODENAME}" = "lunar" ]]; then
+    # There is a python dependency that fails for any older versions of cassandra on ubuntu 23
+    echo "deb http://www.apache.org/dist/cassandra/debian 41x main" | sudo tee -a /etc/apt/sources.list.d/cassandra.sources.list
+
+    # These keys had to be manually added to apt keyring to install java 8 on ubuntu 23
```

Comment:
This comment might need expanding. What does Java 8 have to do with Cassandra 4.1x?

Reply:
Java 17 didn't work with Cassandra in our testing, so we assumed Java 8 was required. Took another look and it appears that Java 11 will be fine.